src/components/WatchLaterCard/WatchLaterCard.jsx

```diff
@@ -0,0 +1,37 @@
+import { useWatchLater } from '../../context/watch-later-context';
+import { thumbnailLink } from '../../utils/video-details-services';
+import './WatchLaterCard.css';
+
+const WatchLaterCard = ({ video }) => {
+    const { _id, title, description, creator } = video;
+    const { removeFromWatchLaterHandler } = useWatchLater();
+
+    const reduceTitleLength = (title) => {
+        if (title.length > 50) return title.substring(0, 47) + '...';
+        return title;
+    };
```

Comment:
Suggestion: You can use CSS ellipsis property to trim your title to a specific number of lines.

Reply:
I have kept it this way so that I can control the character length.